library/func_cpython.py

```diff
@@ -184,9 +184,6 @@ def Init(mem):
     # round()
     # divmod() - probably useful?  Look at the implementation
 
-    # Temporary: use xrange until we rewrite in YSH
-    SetGlobalFunc(mem, 'range', lambda *args: list(xrange(*args)))
```

Comment:
Assuming this comment is out of date based on conversations on #oil-dev that suggest the only way to make ranges now is with `:`. Am I mistaken?